unit_test fails

unit_test failed after building the freshly cloned project:
<pre>
$ make
...
cd tests; TZ=UTC ./unit_test
...
  [0.000] test_user_data
foo 
Expected: {"stack":"    at myFunc (Eval'd code:1)\n    at Eval'd code:1\n    at cfunc\n    at <anonymous> (test_expr:4)\n    at test_expr:3","message":"[d] is not defined"}
Actual  : {"stack":"    at myFunc (Eval'd code:1)\n    at Eval'd code:1\n    at cfunc_0x44f490\n    at <anonymous> (test_expr:4)\n    at test_expr:3","message":"[d] is not defined"}
tests/unit_test.c:4297:1 [ASSERT_EVAL_EQ("var err;\n" "try {\n" "  (function () { \n" "    eval('function myFunc(){d}; myFunc();')\n" "  })()\n" "} catch (e) { err = e; }\n" "err;\n", "({" "'message': '[d] is not defined'," "'stack':\"" "    at myFunc (Eval'd code:1)\\n" "    at Eval'd code:1\\n" "    at cfunc\\n" "    at <anonymous> (test_expr:4)\\n" "    at test_expr:3" "\"})")] (in test_exec_generic)
  [0.005] test_exec_generic
FAIL, run 3660 in 0.132s
make: *** [Makefile:16: test] Error 1
</pre>

`cfunc_0x44f490 <-> cfunc`:
It has to do with the name of an anonymous function, so this seems to be a really minor issue.

--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,7 @@
 WARNS = -W -Wall -pedantic -Wno-comment -Wno-variadic-macros -Wno-unused-function
 V7_FLAGS = -DCS_ENABLE_UTF8
 CFLAGS = $(WARNS) -g -O3 -lm $(V7_FLAGS) $(CFLAGS_PLATFORM) $(CFLAGS_EXTRA)
+TEST_CFLAGS = $(CFLAGS) -DV7_EXPOSE_PRIVATE -DV7_UNIT_TEST -DV7_FILENAMES_SUPPRESS_CFUNC_ADDR
 
 .PHONY: examples test
 
@@ -10,7 +11,7 @@ examples:
 	@$(MAKE) -C examples
 
 tests/unit_test: tests/unit_test.c tests/common/test_util.c tests/common/test_util.h v7.c v7.h Makefile
-		$(CC) tests/unit_test.c tests/common/test_util.c -DV7_EXPOSE_PRIVATE -DV7_UNIT_TEST $(CFLAGS) -lm -o tests/unit_test
+		$(CC) tests/unit_test.c tests/common/test_util.c $(TEST_CFLAGS) -lm -o tests/unit_test
 
 test: tests/unit_test
 	cd tests; TZ=UTC ./unit_test $(TEST_FILTER)
@@ -21,4 +22,3 @@ v7: v7.c v7.h Makefile
 clean:
 	@$(MAKE) -C examples clean
 	rm -fr v7 v7.dSYM tests/unit_test
-

